gamestonk_terminal/etf/etf_controller.py

```diff
@@ -120,29 +114,190 @@ def call_quit(self, _):
         """Process Quit command - quit the program"""
         return True
 
-    def call_web(self, other_args: List[str]):
-        """Process web command"""
-        open_web(other_args)
-
     def call_search(self, other_args: List[str]):
         """Process search command"""
-        name_search(other_args)
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="search",
+            description="Search all available etfs for matching input",
+        )
+        parser.add_argument(
+            "-n",
+            "--name",
+            type=str,
+            dest="name",
+            help="Name to search for",
+            required=True,
+        )
```

Comment:
Can we add Export?

Reply:
I can...but I don't see the purpose?  Same for most of these, it would just be adding it to say we added it.

Reply:
So, I think that all commands that return some sort of data, should have an export command. So that the user can use that data as he wants